user_docs/en/changes.t2t

```diff
@@ -23,8 +23,9 @@ What's New in NVDA
 
 == Changes ==
 - Updated liblouis braille translator to version 3.8.0. (#9013)
-- Add-on authors now can enforce a minimum required NVDA version for their add-ons. Unsupported add-ons will not load or install. (#6275)
-- Add-on authors must now specify the last version of NVDA the add-on has been tested against. Users will be warned against installing or enabling untested add-ons. (#6275)
+- Add-on authors now can enforce a minimum required NVDA version for their add-ons. NVDA will refuse to install or load an add-on whos minimum required NVDA version is higher than the current NVDA version. (#6275)
+- Add-on authors can now specify the last version of NVDA the add-on has been tested against. If an add-on has been only tested against a  version of NVDA lower than the current version, then NVDA will refuse to install or load the add-on. (#6275)
+- This version of NVDA will allow installing and loading of add-ons  that do not yet contain Minimum and Last Tested NVDA version information, but upgrading to future versions of NvDA (E.g. 2019.2) will automatically cause these older add-ons to be disabled.

```

Comment:
Are we sure that this is the case? I assume that, if NVDA 2019.2 won't contain API changes, add-ons without this version info will still load. I'd suggest changing will into may or might.

```suggestion
- This version of NVDA will allow installing and loading of add-ons  that do not yet contain Minimum and Last Tested NVDA version information, but upgrading to future versions of NvDA (E.g. 2019.2) may automatically cause these older add-ons to be disabled.
```